.github/workflows/publish-deploy-staging.yml

```diff
@@ -0,0 +1,125 @@
+name: Publish and deploy on staging
+
+on:
+  push:
+    tags: ["*"]
+  workflow_dispatch:
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set version
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - uses: ./actions/publish-chart
+        env:
+          CHART_PATH: helm-chart/
+          CHART_NAME: renku
+          GIT_USER: chart-bot
+          GIT_EMAIL: chart-bot@example.com
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          GITHUB_TOKEN: ${{ secrets.CI_TOKEN }}
+      - uses: 8398a7/action-slack@v3
```

Comment:
I think it would be nice to use this also on the `deploy.yaml` workflow instead of the bare POST. Having a link to the right commit for example already in the notification is really nice.

Reply:
It's a good point, I'll do that, but in a different PR.

Reply:
https://github.com/SwissDataScienceCenter/renku/pull/2037

Reply:
perfect, thanks!